client/src/components/Filters/FIlterSlider/FilterSlider.js

```diff
@@ -0,0 +1,104 @@
+import React, {Component} from 'react';
+import "./FilterSlider.scss"
+import ReactSlider from 'react-slider'
+import {toast} from 'react-toastify';
+import store from "../../../store";
+import {changeMinPriceFilter, changeMaxPriceFilter} from "../../../action-creators/filter-action-creator";
+
+
+class FilterSlider extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      maxValue: props.maxValue,
```

Comment:
it is not recommended to initiate state with props

Reply:
this props in particular are only seed data, component will not receive this props in the future, this state value will be only controlled inside the component, put props needed to initialize some data without hardcoding it inside the component, which will be not useful in terms of reusability of this component